squidb-addons/squidb-sqlite-bindings/src/com/yahoo/squidb/sqlitebindings/SQLiteBindingsVersion.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2015, Yahoo Inc.
+ * Copyrights licensed under the Apache 2.0 License.
+ * See the accompanying LICENSE file for terms.
+ */
+package com.yahoo.squidb.sqlitebindings;
+
+import com.yahoo.squidb.utility.VersionCode;
+
+/**
+ * Container class for a {@link com.yahoo.squidb.utility.VersionCode} denoting the version of SQLite that is bundled
+ * with the default distribution of the squidb-sqlite-bindings project
+ */
+public final class SQLiteBindingsVersion {
```

Comment:
I think I like it better in SQLiteBindingsAdapter, but if you object I'll revert it.